backend/corpora/common/utils/db_utils.py

```diff
@@ -39,15 +47,50 @@ def _parse_multivalue(value: str) -> typing.List[str]:
         """
         return value.split(",") if value else []
 
-    def query_projects(self):
-        return [
-            {
-                "id": project.id,
-                "status": project.status,
-                "name": project.name,
-                "description": project.description,
-                "processing_state": project.processing_state.name,
-                "validation_state": project.validation_state.name,
+    def get_project(self, key: typing.Tuple[str, str]) -> typing.List[Base]:
+        result = self._query(
```

Comment:
This code seems project specific, so should be moved to `Project`. What is you reasoning for using it in DB?

Reply:
Originally, the reason was to keep DB table logic in DbUtils and keep `Entity` child classes "clean".

Sync'd offline w @Bento007 - agreed to move all project specific logic to `Project` and avoid a `DbUtils` superclass.